pxr/imaging/plugin/hdRpr/python/generateRenderSettingFiles.py

```diff
@@ -121,6 +121,14 @@
                 'defaultValue': 0.0,
                 'minValue': 0.0,
                 'maxValue': 1e6
+            },
+            {
+                'name': 'interactiveMaxRayDepth',
+                'ui_name': 'Interactive Max Ray Depth',
+                'help': 'Controls value of \'Max Ray Depth\' in interactive mode.',
+                'defaultValue': 2,
```

Comment:
Max depth 2 is very low for any glass IMO.

Reply:
So as it’s only active when you move the camera it’s ok, but if it’s a problem for the user exactly for that I expose render setting to control interactive ray depth instead of using constant one